cypress/support/commands.ts

```diff
@@ -1,17 +1,18 @@
 Cypress.Commands.add('getByTestId', (testId: string) => cy.get(`[data-testid="${testId}"]`))
 
 Cypress.Commands.add('login', (email = 'bc-default-0@yopmail.com', password = 'password-0') => {
-  cy.intercept('POST', '/api/auth/callback/credentials').as('login')
+  const uniqLogin = `login-${Date.now()}`
+  cy.intercept('POST', '/api/auth/callback/credentials').as(uniqLogin)
   cy.visit('/login')
   cy.get('[data-testid="input-email"] > .MuiInputBase-root > .MuiInputBase-input').should('be.visible').type(email)
   cy.get('[data-testid="input-password"] > .MuiInputBase-root > .MuiInputBase-input').type(password)
   cy.getByTestId('login-button').click()
-  cy.url().should('eq', `${Cypress.config().baseUrl}/`)
+  cy.wait(`@${uniqLogin}`)
```

Comment:
C'est pas du tout ce qu'on s'etait dit

Reply:
C'est vrai, mais j'ai l'impression que la solution qu'on avait évoqué n'était pas possible dans notre contexte : https://docs.cypress.io/api/commands/wait

En lisant la doc, je comprends qu'il faut que les wait soient tous dans la même fonction (dans le même "it") pour prendre en compte les N occurrences. Avec cette solution, on déclare une requête POST /login unique, et on n'a plus de problème de gestion du nombre d'occurrences. Ensuite, ça permet d'éviter un 3ème paramètre optionnel et de devoir changer beaucoup d'appels aux fonctions login et logout. Ça m'a donc paru être une meilleure solution

Reply:
je sais pas quoi te dire de plus que ce qu'on a evoqué